OpenEXR/IlmImf/ImfMisc.h

```diff
@@ -99,7 +99,7 @@ inline
 int&
 sampleCount(char* base, int xStride, int yStride, int x, int y)
 {
-    char* ptr = base + y * yStride + x * xStride;
+    char* ptr = base + y * ptrdiff_t(yStride) + x * ptrdiff_t(xStride);
```

Comment:
Wouldn't it be better to change the type of the function arguments to ptrdiff_t?

Reply:
I wanted something that could be easily folded into a 2.4.1 (or even backported earlier) without changing any function signatures. I can't wait until 2021 for this fix to proliferate into software that's widely used.

Reply:
But maybe a TODO item to scrub the whole API for places that should be upgraded from 32 to 64 bit ints next time we are willing to have very-breaking changes.